src/modules/reservations.js

```diff
@@ -0,0 +1,45 @@
+import { shows } from '../utils';
+
+const revervations = () => {
+  const closePopupButton = document.getElementById('closePopup');
+  const moviePopup = document.getElementById('moviePopup');
+
+  const titleCards = document.getElementsByClassName('cardboard-title');
+  const reservationsBtn = document.getElementsByClassName('reservations-button');
+
+  const movieTitleElement = document.getElementById('res-movie-title');
+  const movieSummaryElement = document.getElementById('res-movie-summary');
+  const movieGenresElement = document.getElementById('res-movie-genres');
+  const movieScheduleElement = document.getElementById('res-movie-schedule');
+  const movieLanguageElement = document.getElementById('res-movie-language');
+  const movieTvElement = document.getElementById('res-movie-tv');
+  const img = document.getElementById('res-movie-image');
```

Comment:
Optional
- [ ] It might be a good idea to hoist these or pass them as attributes to the functions. That way we can keep our functions pure and therefore easier to test